src/System.Memory/ref/System.Memory.cs

```diff
@@ -488,7 +488,6 @@ namespace System.Text
 {
     public ref partial struct SpanRuneEnumerator
```

Comment:
Is the tool aware of `unmanaged` now or did you have to manually revert that change (to `struct`). Particularly for SequenceReader:
https://github.com/dotnet/corefx/blob/05171beb453577b516caa051ce2bea2ae48d4b83/src/System.Memory/ref/System.Memory.cs#L247

Reply:
I had to revert it manually. It is not yet aware of it unfortunately, but not spending time on fixing it now, however we have: https://github.com/dotnet/arcade/issues/1299